### PR TITLE
update and validate all activities

### PR DESCRIPTION
**Pull request by sanuann**

### CODE REVIEW

File: activities/NDA/items/gender [hunk 1]
```diff
@@ -11,18 +11,16 @@
         "inputType": "radio"
     },
     "responseOptions": {
-        "valueType": "xsd:anyURI",
+        "valueType": "xsd:integer",
```
Comment (satra) [inline activities/NDA/items/gender:14]: why would this change from URI to integer - this is also categorical and finite.

i think the choice should be a URI, so sth like:

```
{ "@id": "http://purl.obolibrary.org/obo/NCIT_C46108",
  "name": "Female (self-reported)",
}
```

and when schema-ui stores the result it should store it in the same way as an id and name

Comment (sanuann) [reply activities/NDA/items/gender:14]: @satra should the above be `@id` or `value`? for the item `sex` we have it like this:
```
"choices": [
            {
                "name": "Female",
                "value": "http://schema.org/Female"
            },
            {
                "name": "Male",
                "value": "http://schema.org/Male"
            },
            {
                "name": "Other",
                "value": "O"
            }
        ]
```

Comment (satra) [reply activities/NDA/items/gender:14]: it should be `value`

**Comment by sanuann** [comment]
the tests fail here because reproschema-py isn't updated. I have tested locally and all tests pass. so merging it for now.

**Pull request merged**